### PR TITLE
Download images for Help Center folder architecture

### PR DESCRIPTION
**Pull request by ricardoaerobr**
[Issue 83](https://github.com/vtexdocs/helpcenter/issues/)

1º Download the file from the original folder (Contentful) to the folder of each article.
2º Update the image reference link inside each article.

### CODE REVIEW

File: docs-utils/update-images.js [hunk 1]
```diff
@@ -22,26 +23,32 @@ const getExtension = (url) => {
...
     if (url.startsWith(baseURL)) return match
 
     let newURL = ''
     if (url.startsWith('http://') || url.startsWith('https://')) {
       const ext = getExtension(url)
-      const filename = `${slug}-${images.length}.${ext}`
-  
+      
+      const newfilepath = `${filepath.split('.')[0]}_${imageIndex}.${ext}`
+      console.log(newfilepath)
```
Comment (PedroAntunesCosta) [inline docs-utils/update-images.js:43]: I think we could get rid of this log here. It has fulfilled its debugging purpose already.

File: docs-utils/update-images.js [hunk 2]
```diff
@@ -22,26 +23,32 @@ const getExtension = (url) => {
 }
 
 const updateImages = async (filepath) => {
+  console.log("Running update images")
+  console.log(filepath)
```
Comment (PedroAntunesCosta) [inline docs-utils/update-images.js:27]: I think we could get rid of this log here. It has fulfilled its debugging purpose already.

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]
I have commented on minor issues, but I would also like to point out that I think we should be mindful of the URLs assembled in the `else if` and `else` conditions.
1. In what situations will they be triggered? It is not clear to me.
2. Given they are triggered, are those the URLs we want to be returned? It seems to me they are not.

File: docs-utils/update-images.js [hunk 3]
```diff
@@ -2,8 +2,9 @@ const fs = require('fs')
 const path = require('path')
 const frontmatter = require('front-matter')
 const imageDownloader = require('image-downloader')
+const { log } = require('console')
```
Comment (PedroAntunesCosta) [inline docs-utils/update-images.js:5]: This import is not used. It can be removed.

File: docs-utils/update-images.js [hunk 4]
```diff
@@ -2,8 +2,9 @@ const fs = require('fs')
...
 
-const baseURL = 'https://raw.githubusercontent.com/vtexdocs/help-center-content/main'
+const baseURL = 'https://raw.githubusercontent.com/vtexdocs/help-center-content/main/'
 const rootDir = path.resolve(__dirname, '..')
```
Comment (PedroAntunesCosta) [inline docs-utils/update-images.js:8]: This variable is not used. It can be removed.

File: docs-utils/update-images.js [hunk 5]
```diff
@@ -27,27 +28,25 @@ const updateImages = async (filepath) => {
   const locale = frontmatter(content).attributes.locale
```
Comment (PedroAntunesCosta) [inline docs-utils/update-images.js:28]: The variables `locale` and `slug` are not used. Their declaration can be removed.

**Review by PedroAntunesCosta — CHANGES_REQUESTED** [review]

File: docs-utils/update-images.js (hunk 5 above)
Comment (ricardoaerobr) [reply docs-utils/update-images.js:28]: Ok, removed.

File: docs-utils/update-images.js (hunk 4 above)
Comment (ricardoaerobr) [reply docs-utils/update-images.js:8]: Ok, removed.

File: docs-utils/update-images.js (hunk 3 above)
Comment (ricardoaerobr) [reply docs-utils/update-images.js:5]: Ok, removed.

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Pull request merged**